docs/transpiler.md

```diff
@@ -111,16 +111,16 @@ then `load()` from your own macro rather than from `@aspect_rules_ts`.
 
 ## Macro expansion
 
-When `transpiler` is used, then the `ts_project` macro expands to these targets:
+When `transpiler` or `declaration_transpiler` is used, then the `ts_project` macro expands to these targets:
```

Comment:
Last nit here. Clarify that `transpiler` and `declaration_transpiler` may be set but _not_ used in some cases, such as when `no_emit` is set. In those cases, the macro will not expand to these targets.